typescript/infra/config/environments/mainnet3/agent.ts

```diff
@@ -295,7 +295,7 @@ const metricAppContexts = [
   },
   {
     name: 'renzo_ezeth',
-    matchingList: routerMatchingList(renzoEzEthAddresses),
+    matchingList: matchingList(renzoEzEthAddresses),
```

Comment:
not using router as the key

Reply:
could we keep tracking the old addresses as `renzo_ezeth_old` below (instead of what's in `renzo_ezeth_old` atm) so we have confidence there are no in-flight messages?

We'll have to be careful rolling it out bc of https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/main/typescript/infra/config/environments/mainnet3/agent.ts#L313, so we should roll it out to rc first, and if we hit that issue we can remove helloworld for now